src/OpenColorIO/ops/fixedfunction/FixedFunctionOpCPU.cpp

```diff
@@ -653,6 +690,145 @@ void Renderer_ACES_DarkToDim10_Fwd::apply(const void * inImg, void * outImg, lon
     }
 }
 
+float compress(float dist, float thr, float scale, float power, float one_over_power)
+{
+    // Normalize distance outside threshold by scale factor
+    const float nd = (dist - thr) / scale;
+    const float p = powf(nd, power);
```

Comment:
There are sometime slight differences between the C and C++ version of these methods (powf in this case) so the code code must now only use the std version.

Reply:
Looks like the whole OCIO code base is using the C version at the moment, it would probably make little sense to just update the code in this PR and not the rest, should we address in a follow up PR ?

Reply:
You pull request is already old enough to avoid adding more changes so I would suggest to change the new code only. The goal is only to avoid 'increasing the problem' for short-term.

Reply:
Sure, will do that now then.